### PR TITLE
Minor issues

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Add confirm modal on mini cart item delete.
Remove top header when currency and locale count is less then 1.
Remove additional information section on empty attribute value.
Fixed image file type attribute issue and transaction date format and amount.
Change customer update route name.

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 
Checked and Found that some issues are fixed and 1 issue is still remaing. 

Issues which are fixed.

1. Cart warning message

Warning

2. Additional Information Hidden.

Additional Feature

3. Header Removed when single locale or currency is presenr for shop.

Header


Attribute issue is still appearing.

**Video**


https://github.com/bagisto/bagisto/assets/92721837/bbaf99aa-de9b-4a00-89a2-97a22c72e5ec

**Comment by shivendra-webkul** [comment]
@ashishkumar-webkul Attribute issue is not appearing. It is working fine please check again.
Reference:- https://webkul.chatwhizz.com/share/view-recording/65ddb0b9095e200579478d8a

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 

Checked the same concern with a fresh setup and then found that the issue has been fixed and working fine. Please check the video for reference.

**Video**


https://github.com/bagisto/bagisto/assets/92721837/ca704620-6852-4672-89fc-1b2c548217d3

File: packages/Webkul/Admin/src/Http/Controllers/Sales/TransactionController.php [hunk 1]
```diff
@@ -127,7 +127,11 @@ public function store(Request $request): JsonResponse
      */
     public function view(int $id)
     {
-        $transaction = $this->orderTransactionRepository->findOrFail($id);
+        $transaction = $this->orderTransactionRepository->findOrFail($id)->toArray();
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Http/Controllers/Sales/TransactionController.php:130]: Remove toArray

File: packages/Webkul/Admin/src/Http/Controllers/Sales/TransactionController.php [hunk 2]
```diff
@@ -127,7 +127,11 @@ public function store(Request $request): JsonResponse
...
+
+        $transaction['created_at'] = date('d-m-Y', strtotime($transaction['created_at']));
+
+        $transaction['amount'] = core()->formatPrice($transaction['amount']);
 
         return new JsonResponse([
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Http/Controllers/Sales/TransactionController.php:136]: Use Transaction Resource.

File: packages/Webkul/Shop/src/Resources/views/products/view.blade.php [hunk 3]
```diff
@@ -7,6 +7,10 @@
     $percentageRatings = $reviewHelper->getPercentageRating($product);
 
     $customAttributeValues = $productViewHelper->getAdditionalData($product);
+
+    $attributeData = collect($customAttributeValues)->filter(function ($item) {
+        return ! empty($item['value']);
+    });
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view.blade.php:13]:
~~~php
collect($customAttributeValues)->filter(fn ($item) => ! empty($item['value']));
~~~

**Review by devansh-webkul — CHANGES_REQUESTED** [review]
Check the above changes.

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 

Please fix all the changes and again assign back for testing.

File: packages/Webkul/Admin/src/Http/Controllers/Sales/TransactionController.php (hunk 2 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Http/Controllers/Sales/TransactionController.php]: Done

File: packages/Webkul/Shop/src/Resources/views/products/view.blade.php (hunk 3 above)
Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Resources/views/products/view.blade.php]: Done

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 

Please check there are some test's which are failed in thr PR, please verify the issues in test case fails.

File: packages/Webkul/Admin/src/Http/Resources/TransactionResource.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+class TransactionResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request
+     * @return array
+     */
+    public function toArray($request)
```
Comment (suraj-webkul) [inline packages/Webkul/Admin/src/Http/Resources/TransactionResource.php:15]: You can use strictly return type of this method and use params like
```php
public function toArray(Request $request): array
{
// ......
}
```

**Review by suraj-webkul — APPROVED** [review]

**Comment by suraj-webkul** [comment]
@shivendra-webkul 

Your test case failing due to routes renaming....

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**